content/docs/2.5/scalers/postgresql.md

```diff
@@ -22,7 +22,7 @@ Alternatively, a user can specify individual
 arguments (host, userName, password, etc.), and the scaler will form a connection string
```

Comment:
Nit - Can you ensure all values use the `PostgreSQL`  casing please given it's a product name?

Reply:
Same for the `<name>:` where the `:` needs to be moved out.

Reply:
Actually let's align with the majority of scalers:
> - `host` - The hostname of the MSSQL instance endpoint.

Reply:
Do you mean using `propery - description`? Prometheus Scaler for example uses `property: description` also. WDYT if in another PR I review the scalers docs and unify the notation?

Reply:
Yes, that notation.

A seperate PR would be awesome, if you are willing to do that of course.

Reply:
Sure, I will do :)

Reply:
I have opened it
https://github.com/kedacore/keda-docs/discussions/539